src/PipelineFramework.Core/PipelineComponentResolvers/DictionaryPipelineComponentResolver.cs

```diff
@@ -0,0 +1,101 @@
+﻿using PipelineFramework.Abstractions;
```

Comment:
Since the IServiceProvider resolver was removed, dedicated resolver folder seems unnecessary.

Reply:
Move resolver and deleted folder